packages/angular/cli/models/parser_spec.ts

```diff
@@ -128,6 +128,7 @@ describe('parseArguments', () => {
     '--e3 true': { e3: true },
     '--e3=true': { e3: true },
     'a b c 1': { p1: 'a', p2: 'b', '--': ['c', '1'] },
+    '-p=1 -c=prod': {'--': ['-p=1', '-c=prod'] },
```

Comment:
You're only adding tests for the last change, not the whole thing. Could you add the following test cases:

- `--p --c`
- `--p 123`
- `--p -c`
- `-p --c`
- `-p --c 123`
- `--c 123 -p`

? See if this works.

Reply:
@hansl, sure thing. I added the requested tests.